### DISCUSSION TITLE
Lftp mirror command downloading 0KB files from source.

### DISCUSSION THREAD

**Discussion by shanksgbs** [lavv17/lftp#694] [Q&A]
@Alexander V. Lukyanov 

Lftp mirror command downloading 0KB files from the source.

lftp -v
LFTP | Version 4.4.8 | Copyright (c) 1996-2013 Alexander V. Lukyanov

Operating system Redhat Linux

Tried below options in abash script
mirror -n -c $REMOTEDIR $LOCALDIR

As per some articles and suggestion Alexander latest version for RedHat is installed
 lftp -v
LFTP | Version 4.5.3 | Copyright (c) 1996-2014 Alexander V. Lukyanov

Now I am getting **mirror: x.csv: file size decreased during transfer**

Would you be able to share your suggestion to solve this?

**Comment by kk-hiraskar** [comment]
This happening on Linux 8..
On Linux 7 same command working fine..
Not sure how to fix this..
